### PR TITLE
Grid design

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary
- Change method for design to CSS Grid

## Screenshot
dec-14-2018 11-35-59

## Trello Card
https://trello.com/c/DqEKRQeh/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: scss/tech-grid.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  grid-template-areas: "first-tech others-tech others-tech"
+                      "first-tech others-tech others-tech";
+  justify-items: center;
+
+  .tech-item:first-child {
+    border-right: solid 2px $green;
+    grid-area: first-tech;
+  }
+
+  .tech-items:not(.tech-items:first-child) {
```
Comment (CarlosHWolox) [inline scss/tech-grid.scss]: 0_0 wow

**Review by CarlosHWolox — APPROVED** [review]

File: scss/tech-grid.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.tech-container {
+  align-items: center;
+  align-self: center;
+  display: grid;
+  grid-template-columns: 250px 250px 250px;
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: repeat(3, 250px);

File: scss/tech-grid.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  grid-template-rows: 1fr 1fr;
+  grid-template-areas: "first-tech"
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: this is not necesary

File: scss/tech-grid.scss [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                      "first-tech";
+  justify-items: center;
+
+  .tech-item:first-child {
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: why not using &?

File: scss/tech-grid.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    border-right: solid 2px $green;
+    grid-area: first-tech;
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: remove this

File: scss/tech-grid.scss (hunk 4 above)
Comment (FrankIglesias) [reply scss/tech-grid.scss]: Use grid-row

File: scss/tech-grid.scss [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  }
+}
+
+.tech-item {  
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  padding: 30px 35px 30px 35px;
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: use /2 shorthand

File: scss/tech-grid.scss [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  text-align: center;
+}
+
+.tech-logo {
+  margin-bottom: 10px;
+  height: 100px;
```
Comment (FrankIglesias) [inline scss/tech-grid.scss]: set only one of the sizes

File: scss/styles.scss [hunk 8]
```diff
@@ -5,7 +5,9 @@
 .body {
   margin: 0 auto;
   max-width: 1000px;
-  width: 90%
+  width: 90%;
```
Comment (FrankIglesias) [inline scss/styles.scss]: Why this width?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/tech-grid.scss (hunk 3 above)
Comment (eledwinn) [reply scss/tech-grid.scss]: grid-template-rows removed.

File: scss/tech-grid.scss (hunk 2 above)
Comment (eledwinn) [reply scss/tech-grid.scss]: Ok.

File: scss/tech-grid.scss (hunk 6 above)
Comment (eledwinn) [reply scss/tech-grid.scss]: Ok.

File: scss/tech-grid.scss (hunk 1 above)
Comment (eledwinn) [reply scss/tech-grid.scss]: Removed! xD

File: scss/styles.scss (hunk 8 above)
Comment (eledwinn) [reply scss/styles.scss]: Main content (<body>) centered with flex.

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**